PWGCF/JCorran/Tasks/CMakeLists.txt

```diff
@@ -13,3 +13,13 @@ o2physics_add_dpl_workflow(jfluc-analysis
                     SOURCES jflucAnalysisTask.cxx
                     PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::JCorran
                     COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(flow-spc-task
+                    SOURCES flowJSPCAnalysisTask.cxx
```

Comment:
Please, do not use the name task neither in the executable not in the source filename

Reply:
Well, the task files are in the `Tasks` directory and the class files should go into the `Core` directory

Reply:
Alright, I renamed them now.